dcos_test_utils/dcos_cli.py

```diff
@@ -155,8 +155,16 @@ def setup_enterprise(
              "--username={}".format(username), "--password={}".format(password)])
         assert stdout == ''
         assert stderr == ''
-        self.exec_command(
-            ["dcos", "package", "install", "dcos-enterprise-cli", "--cli", "--yes"])
+
+        if '1.11' in DCOS_CLI_URL or '1.10' in DCOS_CLI_URL:
```

Comment:
I suggest:        
        cmd = ["dcos", "package", "install", "dcos-enterprise-cli", "--cli", "--yes"]
        if '1.11' in DCOS_CLI_URL or '1.10' in DCOS_CLI_URL:
            # An attempt to work around
            # https://jira.mesosphere.com/browse/DCOS-45738
            cmd.append('--global')
        self.exec_command(cmd)